Libraries/Opc.Ua.Gds.Server.Common/GlobalDiscoverySampleServer.cs

```diff
@@ -71,6 +75,12 @@ protected override void OnServerStarted(IServerInternal server)
         {
             base.OnServerStarted(server);
 
+            //ToDo delete this code in a production environment as this creates hardcoded passwords
+            if (m_createStandardUsers)

```

Comment:
we need to look at this...

Reply:
yeah, I added this to be compatible with existing implementations, maybe we can default the field to false in future versions to comply to security by default.